.eslintrc.js

```diff
@@ -398,13 +421,11 @@ module.exports = {
       },
     },
     {
-      files: ['ui/components/multichain/**/*.{js,ts,tsx}'],
+      files: ['ui/components/multichain/**/*.{js}'],
```

Comment:
According to a comment in the PR that introduced this rule set (https://github.com/MetaMask/metamask-extension/pull/21833), it didn't work on typescript files (but included them anyway), so it appears that we don't need it (and it'd fail if you tried because of other breaking changes).